pkg/config/config_template.yaml

```diff
@@ -217,7 +218,40 @@ api_key:
 #
 # The timeout to execute the command in second
 # secret_backend_timeout: 5
-
+#
+{{ end -}}
+{{- if .LogsAgent }}
+# Logs agent
+#
+# Enable logs collection for all containers, disabled by default
```

Comment:
`logs_enabled` is to enable/disable the Logs Agent entirely, not related to logs collection for all containers

Reply:
updated